style/style.css

```diff
@@ -0,0 +1,312 @@
+.homepage-products img {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+}
+
+
+footer{
+    grid-area: footer;
+    background-color: rgba(194, 195, 200, 0.35);
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    padding: 1rem;
+    gap: .5rem;
+}
+
+footer span {
+    border-left: 3px solid rgba(94, 92, 92, 0.532);
+    padding: .3rem;
+    width: 40%;
+    color:rgb(33, 33, 26);
+    background-color: rgba(169, 169, 169, 0.172);
+
+}
+
+.copy{
+    text-align: center;
+    color: rgb(49, 49, 49);
+    font-weight: 900;
+}
+
+/* Table mode */
+
+@media screen and (min-width: 540px){
+    body {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        grid-template-rows: 10rem auto auto auto;
+        grid-template-areas: 
+        "hd hd hd hd"
+        "nav nav nav nav"
+        "section section section section"
+        "footer footer footer footer";
+        /* grid-row-gap: 1rem; */
+    }
+
+    .header {
+        grid-area: hd;
+    }
+
+    .add {
+        max-width: 100%;
+        word-wrap: break-word;
+    }
+    
+    .add-text{
+        font-size: 1rem;
+        word-wrap: break-word;
+    }
+
+    .nav2 {
+        grid-area: nav;
+        display: flex;
+        justify-content: flex-end;
+        align-content: flex-end;
+        /* background-color: rgb(174, 255, 0); */
+    }
+
+    .nav-body{
+        width: 88%;
+        background-color: rgba(194, 195, 200, 0.35);
+        display: flex;
+        justify-content: flex-end;
+        align-content: flex-end;
+        margin-right: 2rem;
+    }
+
+    .nav-li{
+        padding: 0.5rem 0;
+        /* padding: 1rem; */
+        display: flex;
+        grid: 5rem;
+        text-decoration: none;
+        list-style: none;
+        margin-right: 1rem;
+    }
+
+    .nav-li li{
+        padding: auto;
+        margin-left: 1rem;
+        font-size:xx-small;
+        color: #5c0658;
+        font-weight: 1000;
+
+    }
+
+    section {
+        grid-area: section;
+        grid-template-columns: repeat(4, 1fr);
+        grid-template-rows: repeat(2, auto);
+        grid-template-areas: 
+        "wimg wimg hero hero"
+        "cakes cakes cakes cakes";
+        margin: 1rem;
+    }
+
+    .wrapper-img {
+        background-color: rgba(194, 195, 200, 0.35);
+        grid-area: wimg;
+        border-left: 1rem solid rgba(194, 195, 200, 0.35);
+        border-right: 1rem solid rgba(194, 195, 200, 0.35);
+
+    }
+
+    .wrapper-img .img-hero {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+    }
+
+    .img-hero img{
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+    }
+
+    .main-hero {
+        grid-area: hero; 
+        margin: 0;   
+        }
+    
+    .title{
+        margin-top: 1rem;
+        font-size: 1.5rem;
+        text-align: right;
+        font-weight: 900;
+    }
+
+    .text-hero{
+        text-align: right;
+        font-size: smaller;
+        padding: auto;
```

Comment:
I don't think this is valid. You can see styles that are invalid / not used by looking at Chrome devtools, in the Elements panel select this element and look in the Styles panel
![image](https://user-images.githubusercontent.com/471250/202225475-dadf639a-0002-464b-937e-5c239bbb8468.png)